packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -658,6 +662,9 @@ abstract class _BuildIOSSubCommand extends BuildSubCommand {
       configOnly: configOnly,
       buildAction: xcodeBuildAction,
       deviceID: globals.deviceManager?.specifiedDeviceId,
+      disablePortPublication: usingCISystem &&
+          xcodeBuildAction == XcodeBuildAction.build &&
+          await disablePortPublication,
```

Comment:
@christopherfujino I made an update to this, so it only checks `disablePortPublication` if the action is build. I did this because I didn't add the `publish-port` flag to the `build ipa` (archive) command, which also uses this function.

Original PR failure: https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_ios%20ios_content_validation_test/13690/overview

New PR passing: https://chromium-swarm.appspot.com/task?id=655c94b693aaad10